xylog/README.md

```diff
@@ -190,6 +194,21 @@ logger.Critical("foo foo")
 // CRITICAL foo foo
 ```
 
+## EventLogger
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/xybor/xyplatform/pullRequest?prid=10050221)
```suggestion
## EventLogger

```